src/listeners/notifications/streamComment.ts

```diff
@@ -14,6 +14,14 @@ export class NotificationListener extends Listener<typeof AmanekoEvents.StreamCo
 			where: { channelId, relayChannelId: { not: null } },
 			select: { relayChannelId: true }
 		});
+
+		const guildsYoutubeBlacklist = (
```

Comment:
You can probably move this to after the `relaysChannelIds` length check.

Also, I wonder if you could instead change the subscription query to add something like:
```ts
where: {
    // ...
    guild: {
	    blacklist: {
		    none: { channelId: comment.channel_id }
	    }
    }
}
```